python_utils/stack-pt-merge.py

```diff
@@ -619,25 +557,26 @@ def iter_days():
             if not stack_groups_only:
 
                 # Add variables from inln_mole file
-                blist = list(b.variables.keys())
-                blist.remove("TFLAG")
-                for vn in blist:
-                    v = b.variables[vn]
+                for vn in [
+                    vn
+                    for vn in list(a.variables.keys())
+                    if vn not in sg_vns
+                ]:
+                    v = a.variables[vn]
                     if vn not in vars_:
                         vars_[vn] = c.createVariable(vn, np.float32, ("time", POINT_DIM_NAME),
-                            zlib=True, complevel=4, fill_value=0.
+                            zlib=True, complevel=1, fill_value=0.
```

Comment:
I think `fill_value=0` could be removed. We are filling with 0 below, and every time-point should be set from the input data anyway.

Reply:
2e3af63